request_handlers_test.cc

```diff
@@ -0,0 +1,111 @@
+#include "request_handlers.h"
+#include "gtest/gtest.h"
```

Comment:
Can we reorder these includes to be alphabetical? I totally agree that the gtest header is less important, but this just maintains a consistent style.